codemods/v2/borderedlist.ts

```diff
@@ -2,38 +2,72 @@
 import { FileInfo, API, Options } from 'jscodeshift';
 import { Codemod } from '../helpers';
 
-module.exports = function autocompleteProps(
-  fileInfo: FileInfo,
-  api: API,
-  options: Options,
-): string | null | undefined | void {
-  const mod = new Codemod(fileInfo, api);
+function addBorderedProp(mod: Codemod, node) {
+  const hasBorderedProp = node.openingElement.attributes.filter(
+    attr =>
+      attr.type === 'JSXAttribute' &&
+      attr.name.type === 'JSXIdentifier' &&
+      attr.name.name === 'bordered',
+  );
+
+  if (hasBorderedProp.length === 0) {
+    const borderedProp = mod.createNode(cs => cs.jsxAttribute(cs.jsxIdentifier('bordered')));
+    node.openingElement.attributes.push(borderedProp);
+  }
+}
+
+function addBorderedPropToItems(mod: Codemod, name: string) {
+  mod.source.find(mod.cs.JSXElement).forEach(({ node }) => {
+    if (
+      node.openingElement.name.type === 'JSXIdentifier' &&
+      node.openingElement.name.name === name
+    ) {
+      addBorderedProp(mod, node);
+    }
+  });
+}
 
+function addBorderedPropToChildren(mod: Codemod, name: string) {
   mod.source.find(mod.cs.JSXElement).forEach(({ node }) => {
     if (
       node.openingElement.name.type === 'JSXIdentifier' &&
-      node.openingElement.name.name === 'BorderedList'
+      node.openingElement.name.name === name
     ) {
       node.children.forEach(child => {
         if (
           child.type === 'JSXElement' &&
           child.openingElement.name.type === 'JSXIdentifier' &&
           child.openingElement.name.name === 'Item'
         ) {
-          const borderedProp = mod.createNode(cs => cs.jsxAttribute(cs.jsxIdentifier('bordered')));
-          child.openingElement.attributes.push(borderedProp);
+          addBorderedProp(mod, child);
         }
       });
+    }
+  });
+}
+
+module.exports = function autocompleteProps(
+  fileInfo: FileInfo,
+  api: API,
+  options: Options,
+): string | null | undefined | void {
+  const mod = new Codemod(fileInfo, api);
 
-      node.openingElement.name.name = 'List';
+  // Add `bordered` to `BorderedListItem`
+  ['@airbnb/lunar/lib/components/BorderedList/Item'].forEach(importPath => {
```

Comment:
Probably need to support `import { Item as FooItem } from '@airbnb/lunar/lib/components/BorderedList';` as well.

Reply:
resolving! going to leave it as `FooItem` or however they've renamed the import. `Item` doesn't need a rename